package.json

```
@@ -34,9 +34,13 @@
     "compile:types": "tsc --emitDeclarationOnly --declaration",
     "compile:browser": "webpack --progress --env target=web",
     "docs": "rimraf docs && typedoc",
-    "test": "npm run test:node && npm run test:browser",
-    "test:browser": "npm run compile:browser && jest --verbose --config=jest.config.ts --selectProjects=dom",
-    "test:node": "jest --verbose --config=jest.config.ts --selectProjects=node",
+    "test": "npm run compile:browser && jest --verbose --config=jest.config.ts --selectProjects=node:unit node:integration dom:integration",
```

I think it is more straightforward with less error possibility.

```suggestion
    "test": "npm run test:unit && npm run test:integration",
```

---

I am against this. The whole idea behind using the custom `testSequencer` was so all tests would run in one process to speed up the execution, for example avoiding double test discovery etc. This would execute the tests in two processes.